### ISSUE TITLE
Pipe in PCM data from node stream?

### ISSUE THREAD

**Issue by Xeonzinc** [geekuillaume/node-audioworklet#1]
Hi, I have an application which currently mixes PCM data into a node stream. I've trialed a few other node audio outputs which all allow for piping of audio straight to the output device. This package seems like it solves some shortcomings with others, but I'm not sure how to convert a 'pipe' into the equivalent processFrame function which is needed here, is this possible in some way?

**Comment by geekuillaume** [comment]
Yes you can do this. You need to `[YOUR_STREAM].read(length)` in the process function and copy this data in the TypedArray passed as first argument to the process function. The `lenght` argument is in bytes so it should be equal to `samples count * channels count * bytes per sample (4 if using float32, 2 if using signed 16 int)`.
